src/RegistrationController.js

```diff
@@ -172,6 +184,9 @@ function (
       // register parse complete event listener
       self.state.get('schema').on('parseComplete', function(updatedSchema) {
         var modelProperties = updatedSchema.properties.createModelProperties();
+        if (!updatedSchema.error) {
```

Comment:
Not sure if we need this if check as we already are checking for `updatedSchema.error` below on line 200.

Reply:
this is needed for a unit test that does not have policy populated

Reply:
seems odd that we are making a code change for the test. Have we already tried to mock the server response?

Reply:
hmm.. my bad.. testing it again after removing this check and all tests pass. I have removed this check